main.py

```diff
@@ -605,6 +605,23 @@ class Me:
     if os.path.exists("gsheetsadmin/{}".format(deprecationFile)):
         print(f"{bcolors.WARNING}Authentication via {deprecationFile} is deprecated. Consider removing this file and using sstimmy.json instead.{bcolors.ENDC}")
 
+async def _on_message_(message: discord.Message):
+    if not message.guild.id == 763119924385939498:
+        return
+    if not message.author.id == 752984497259151370:
+        return
+    await message.add_reaction(discord.PartialEmoji(name="TurtleSmirk", id=957074334084644874))
```

Comment:
raw id

Reply:
Use core.common and add it in there

Reply:
fixed, im not making configcat for my user id 😭